src/commands/kv/key/put.rs

```diff
@@ -13,13 +13,24 @@ use crate::http;
 use crate::settings::global_user::GlobalUser;
 use crate::settings::toml::Target;
 use crate::terminal::message::{Message, StdOut};
+use reqwest::blocking::multipart;
+use reqwest::blocking::Body;
+
 pub struct KVMetaData {
     pub namespace_id: String,
     pub key: String,
     pub value: String,
     pub is_file: bool,
     pub expiration: Option<String>,
     pub expiration_ttl: Option<String>,
+    pub metadata: Option<serde_json::Value>,
+}
+
+// to be used with clap's Arg::validator
+pub fn metadata_validator(arg: String) -> Result<(), String> {
+    serde_json::from_str::<serde_json::Value>(&arg)
+        .map(|_| ())
+        .map_err(|e| e.to_string())
```

Comment:
The error serde gives isn't very helpful.  It's going to be especially confusing because as things stand

    --metadata '"somestring"'

will work, but not
    
     --metadata "'somestring'"
     --metadata "somestring"

If we cant fix that, we should at least make the error more helpful

Reply:
I added something to help out here. I think it's good to require the double quotes so that we maintain a sensible input space.

Reply:
Yeah, totally.

I think it's probably better to use a fixed valid example string in the error message, rather than their input, because adding quotes to their input won't necessarily make it valid.

Otherwise LGTM thanks

Reply:
done